src/storage/sequelize.ts

```diff
@@ -142,8 +147,8 @@ export class SequelizeStorage implements UmzugStorage {
 				tableName: this.tableName,
 				schema: this.schema,
 				timestamps: this.timestamps,
-				charset: 'utf8',
-				collate: 'utf8_unicode_ci',
+				charset: DIALECTS_WITH_CHARSET_AND_COLLATE.includes(this.sequelize.dialect.name) ? 'utf8' : null,
+				collate: DIALECTS_WITH_CHARSET_AND_COLLATE.includes(this.sequelize.dialect.name) ? 'utf8_unicode_ci' : null,
```

Comment:
```suggestion
				charset: DIALECTS_WITH_CHARSET_AND_COLLATE.includes(this.sequelize.dialect!.name!) ? 'utf8' : null,
				collate: DIALECTS_WITH_CHARSET_AND_COLLATE.includes(this.sequelize.dialect!.name!) ? 'utf8_unicode_ci' : null,
```

These are always given in the actual sequelize code, just not in our simple mock type